logging/src/logging.cc

```diff
@@ -168,6 +169,8 @@ namespace leatherman { namespace logging {
 
         if (use_event_log) {
             log_eventlog(level, message);
+        } else if (use_syslog) {
+            log_syslog(level, message);
         } else {
```

Comment:
maybe we should replace use_event_log/use_event_log flags with logging_backend enum and make the dispatch using switch

Reply:
we could also move boost related logging to `log_boost`